backend/routes.js

```diff
@@ -258,6 +267,21 @@ export default class Servers {
                         status = 201;
                     }
                     saveSetting('USERS', existingUsers);
+
```

Comment:
similiarly to https://github.com/plotly/plotly-database-connector/pull/180/files#r141751154, it looks like we're missing the check that verifies that this user is allowed to view this app